mmv1/third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -1457,6 +1457,14 @@ func resourceContainerCluster() *schema.Resource {
 					},
 				},
 			},
+<% unless version == 'ga' -%>
+			"auto_provisioning_network_tags": {
```

Comment:
In general we try to keep new fields close to the API field names so that users can cross-reference with the API docs if needed - so in this case that would mean calling this field `node_pool_auto_config`. I hear you that it's a little unclear since the field is used for multiple features - but at least we would be consistently inconsistent :-)

Reply:
Sure, I think keeping it closer to the API name makes sense. I'll see if I can put something together today